src/ports/postgres/modules/recursive_partitioning/random_forest.sql_in

```diff
@@ -249,9 +250,13 @@ forest_train(training_table_name,
       </tr>
       <tr>
         <th>tree</th>
-        <td>BYTEA8. Trained tree model stored in binary 
+        <td>BYTEA8. Trained tree model stored in binary
         format (not human readable).</td>
       </tr>
+      <tr>
+        <th>impurity_var_importance</th>
+        <td>DOUBLE PRECISION[]. The gini impurity importance score for the tree.</td>
```

Comment:
Also mention the order corresponds to the order of the variables as found in    'cat_features' followed by the order of the 'con_features' in summary table